From 556836a968936258bc9f87a982c7f50e46597ef2 Mon Sep 17 00:00:00 2001
From: TechnoPorg <mail@technop.org>
Date: Mon, 1 Dec 2025 18:33:52 -0500
Subject: [PATCH] Disable arch feature in libm for now

---
 Cargo.toml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index 2dc2a76..8484405 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -40,5 +40,8 @@ once_cell = "1"
 lru = "0.14"
 ahash = ">=0.8.9"
 
+[patch.crates-io]
+libm = { git = "https://github.com/TechnoPorg/compiler-builtins.git", rev = "304529647400242739f304a63ad76b463a084614" }
+
 [profile.bench]
 debug = true
-- 
2.52.0

